src/js/control-bar/progress-control/progress-control.js

```diff
@@ -157,6 +157,9 @@ class ProgressControl extends Component {
    */
   handleMouseDown(event) {
     const doc = this.el_.ownerDocument;
+    const seekBar = this.getChild('seekBar');
```

Comment:
Just in case the seekBar was removed from the player through the player options, you'll want to double check the seekBar component was found before calling `seekBar.handleMouseDown()`

Reply:
That's true, I have also checked other function in this component to make sure seekBar exist.